docs/payment.md

```diff
@@ -84,7 +84,7 @@ class Item extends Model implements ProductLimitedInterface
 ```
 
 I do not recommend using the limited interface when working with a shopping cart.
-If you are working with a shopping cart, then you should override the `PurchaseServiceInterface` interface.
+If you are working with a shopping cart, then you should override the `ProductLimitedInterface` with the `PurchaseServiceInterface` interface.
```

Comment:
I will answer here immediately for all three PRs. It is not correct. You are confusing the service contract and the model contract. 

This talks about the public contract of the service that works with purchases. The service has one single method that returns a list of transfers for making a refund.

https://github.com/bavix/laravel-wallet/blob/a83b72acaf946557dd15d6df60fa2481d4d8c8fd/src/Traits/CartPay.php#L201-L202

Reply:
add how to override this in the documentation will be of great help. Or documenting what the `PurchaseServiceInterface` actually does

Reply:
The laravel documentation answers your question:
https://laravel.com/docs/10.x/container#binding

No one asks about it. There are two options: everything is clear or no one uses it. I'm sticking with the second option.